src/common/DOMHelper.js

```diff
@@ -0,0 +1,3 @@
+export const $ = (selector) => document.querySelector(selector);
+
+export const $$ = (selector) => document.querySelectorAll(selector);
```

Comment:
깔끔하네요:) 이렇게도 확장할 수 있습니다. 
```suggestion
export const $ = (selector, node = document) => node.querySelector(selector);
export const $$ = (selector, node = document) => node.querySelectorAll(selector);
```

Reply:
다른 요소가 base가 되는 것을 추가로 구현할 수 있겠군요. 감사합니다😀

[수정 commit](https://github.com/al-bur/javascript-racingcar/commit/94a903e7587e14a92e96321349dad5c71719a73e)